assets/element-templates.css

```diff
@@ -98,3 +98,7 @@
 .bio-properties-panel-deprecated-template-text {
   width: 216px;
 }
+
+.bio-properties-panel-header-link {
+  color: var(--link-color);
```

Comment:
Where is `--link-color` defined? Can we safely re-use it here? What I'm wondering is if we define the color in the right place.

I'd assume:

* The integrating application (i.e. Camunda Modeler) defines a link color, this library will follow suite (will properly color links, and hence the documentation link, too).

Why is it not happening in this case?

Reply:
As we discussed, two options: 

* Generalize the properties-panel styles, all standard links should have a standard color
* Move link styling to desktop modeler (entirely); there is little value in mandating a style inside a library; the application should define link colors, and the library should _follow suite_.